src/gigs-board/pages/community/projects.jsx

```diff
@@ -236,10 +246,65 @@ const CommunityProjectsPage = ({ handle }) => {
                     adornment: "bi bi-tools",
                   },
                   label: "Create project",
-                  onClick: () => {},
+                  onClick: () => {
+                    State.update({
+                      createProject: !state.createProject,
+                    });
+                  },
                 })
               : null}
           </div>
+          <div className="d-flex align-items-center justify-content-center">
+            {state.createProject &&
+              widget("components.organism.editor", {
+                classNames: {
+                  submit: "btn-primary",
+                  submitAdornment: "bi-check-circle-fill",
+                },
+
+                heading: "Create project",
+                isEditorActive: state.isEditorActive,
+                isEditingAllowed: Viewer.can.editCommunity,
+                onCancel: () => State.update({ createProject: false }),
+                onChangesSubmit: createProject,
+                submitLabel: "Accept",
+                data: state.teamData,
+                schema: {
+                  name: {
+                    inputProps: {
+                      min: 2,
+                      max: 30,
+                      placeholder: "Project name",
+                      required: true,
+                    },
+
+                    label: "Name",
+                    order: 1,
+                  },
+                  description: {
+                    label: "Description",
+                    order: 2,
+                    inputProps: {
+                      min: 2,
+                      max: 60,
+                      placeholder:
+                        "Describe your project in one short sentence that will appear below the tile of the project on this page.",
```

Comment:
I think it's ok to shorten this to just:

```suggestion
                        "Describe your project in one short sentence.",
```